tokio-trace/Cargo.toml

```diff
@@ -30,6 +31,7 @@ ansi_term = "0.11"
 humantime = "1.1.1"
 futures = "0.1"
 log = "0.4"
+criterion = { version = "0.2", default_features = false }
```

Comment:
I'm not _opposed_ to using `criterion` for `tokio-trace`'s benchmarks, but I'd like the motivation to be recorded in the PR description. Also, I think a change to switch to `criterion` should be moved to a separate PR, since we tend to merge PRs by squashing them --- since it's not directly related to the rest of the changes in this PR, it should have its own own commit in the master git history.

Reply:
I can revert the change; just FYI though, the benchmarks aren't compiling on current nightly due to missing `dyn` annotations. I can add those in, but I wasn't sure if it would screw with backwards-compatibility requirements.

Reply:
We should probably add an `allow` attribute to fix the benchmarks on the latest nightly, but they don't get built on the CI builds that run against earlier Rust versions (since those are stable), so it would probably also be fine to just add `dyn`. 

Again, I'm fine with switching to `criterion` (and being able to run the benches without needing a nightly compiler would make my life easier, too 😄) but we should probably do it in a separate branch. Sorry for being persnickety, I just like having a clean git history on master.

Reply:
Thanks for moving this change to #1163.